data/kahet/kahet missions.txt

```diff
@@ -137,27 +137,27 @@ mission "graveyard insights compatibility"
 
 event "graveyard insights"
 	planet Alix
-		add description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, station Parii'het would have run out of supplies thousands of years ago.`
+		add description `	 Here the Ka'het attacks were limited to the inhabited settlements; had they also destroyed the mining bases and the unmanned vessels, Parii’het would have run out of supplies thousands of years ago.`
 	planet Celeatis
 		add description `	This planet appears to have been home to one of the largest Builder research centers, which focused on studying the anomaly in the system. Had they found out how to stabilize it and use it as a wormhole, perhaps their species would have survived and spread to the rest of the Ember Waste.`
 	planet Ceilia'sei
-		add description `	The main purpose of Ceilia'sei was to act like some sort of anti-missile platform. But when the Ka'het invasion broke out for the second time, it had no way to defend deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
+		add description `	Ceilia’sei’s main purpose was to act as some sort of anti-missile platform, but when the Ka’het invaded a second time, the station had no way to defend against their deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
```

Comment:
```suggestion
		add description `	The main purpose of Ceilia'sei was to act as some sort of anti-missile platform, but when the Ka'het invaded a second time, the station had no way to defend against their deadly Ravager Beams, and the station's defense systems were subsequently lost forever.`
```

I'd suggest switching this bit back. It's not best form, but it's not incorrect either, and it avoids making the name more complex.